engines/drascula/interface.cpp

```diff
@@ -22,6 +22,75 @@
 #include "drascula/drascula.h"
 #include "graphics/cursorman.h"
 
+#include "common/config-manager.h"
+#include "common/text-to-speech.h"
+
+// The verbs are represented in-game as a picture, thus we are
+// adding transcriptions here
+// While only English, Spanish, Italian, and Russian are translated in-game,
+// they are translated for the TTS system here
+static const char *verbNamesEnglish[] = {
+	"Walk",
+	"Look",
+	"Take",
+	"Open",
+	"Close",
+	"Talk",
+	"Push"
+};
+
+static const char *verbNamesSpanish[] = {
+	"Ir a",
+	"Mirar",
+	"Coger",
+	"Abrir",
+	"Cerrar",
+	"Hablar",
+	"Mover"
+};
+
+static const char *verbNamesItalian[] = {
+	"Vai",
+	"Guarda",
+	"Prendi",
+	"Apri",
+	"Chiudi",
+	"Parla",
+	"Premi"
+};
+
+static const char *verbNamesFrench[] = {
+	"Marcher",
+	"Regarder",
+	"Ramasser",
+	"Ouvrir",
+	"Fermer",
+	"Parler",
+	"Pousser"
+};
+
+static const char *verbNamesGerman[] = {
+	"Gehe",
+	"Schau",
+	"Nimm",
+	"Oeffne",
+	"Schliesse",
+	"Rede",
+	"Druecke"
+};
+
+static const char *verbNamesRussian[] = {
+	"Идти",
+	"Смотреть",
+	"взять",
+	"Открыть",
+	"Закрыть",
+	"Говорить",
+	"Толкать"
```

Comment:
I am commenting here, but this is valid for any non-ASCII characters in other languages as well:
- They need to be in the encoding you use when calling `ttsMan::say()`. Here you don't pass an encoding, so that means UTF-8.
- To avoid relying on the `interface.cpp` encoding it is better to represent all non-ASCII character with their octal or hexadecimal value. See for example how it is done for the text in [scummvm/devtools/create_drascula/staticdata.h](https://github.com/scummvm/scummvm/blob/master/devtools/create_drascula/staticdata.h) (starting around line 920).